ansible/pbTestScripts/buildJDK.sh

```diff
@@ -77,7 +77,26 @@ checkJDKVersion() {
                         JAVA_TO_BUILD="jdk14u";;
                 *)
                         echo "Not a valid JDK Version" ; showJDKVersions; exit 1;;
-        esac
+	esac
+	setBootJDK 
+}
+
+setBootJDK() {
+	local buildJDKNumber=$(echo ${JAVA_TO_BUILD//[!0-9]/})
+	local bootJDKNumber=$(($buildJDKNumber - 1));
+        # if building JDK8/9 look for 'jdk8u', not 'jdk-x'
+	if [[ $buildJDKNumber -eq 8 || $buildJDKNumber -eq 9 ]]; then
+		export JDK_BOOT_DIR=$(find /usr/lib/jvm -maxdepth 1 -name *jdk8*)	
```

Comment:
You seem to have added a fixed `jdk8` search immediately after checking if it's either 8 or 9. To be honest, I'd ignore 9 entirely now

Reply:
I did that specifically because the playbooks don't install `jdk9`, so the only other option to build jdk9 is going to be jdk8 - but fair enough - I know it's out of support but I thought if the option to make it able to build is there, we may as well put it in :-)